sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -134,8 +134,9 @@ def begin_recognize_receipts_from_url(self, url, **kwargs):
         """Extract field text and semantic values from a given US sales receipt.
         The input document must be the location (Url) of the receipt to be analyzed.
 
-        :param url: The url of the receipt. Currently only supports US sales receipts.
-        :type url: str
+        :param url: The url of the receipt to analyze. The input must be a valid, encoded url
```

Comment:
it should either be ':param str url:' or add ':type url: str'

Reply:
Thanks for catching! I accidentally deleted types when updating the url docstrings